common/flatpak-dir.c

```diff
@@ -9677,7 +9801,19 @@ flatpak_dir_get_locale_subpaths (FlatpakDir *self)
       if (flatpak_dir_is_user (self))
         subpaths = flatpak_get_current_locale_subpaths ();
       else
-        subpaths = g_new0 (char *, 1);
+        {
+          GDBusProxy *proxy = get_accounts_dbus_proxy ();
+          if (proxy)
+            {
+              subpaths = get_locale_subpaths_from_accounts_dbus (proxy);
```

Comment:
I don't know how AccountService works though. If you have an ldap user database configured, it could easily have 50000 users. Are those exported via the dbus api?